topology/Georgia Institute of Technology/Georgia Tech/Georgia Tech PACE.yaml

```
@@ -1,9 +1,9 @@
 GroupDescription: Resources in the Georgia Tech PACE cluster, to be used by LIGO.
 GroupID: 423
-Production: true
+Production: false
```

```suggestion
Production: true
```
"Production" refers to the grid group; "true" means it's part of the production (default) grid, "false" means it's part of the ITB (testing) grid. So changing this is not necessary.